frontend/appflowy_flutter/integration_test/desktop/database/database_row_test.dart

```diff
@@ -50,7 +49,6 @@ void main() {
 
       // The initial number of rows is 3
       await tester.assertNumberOfRowsInGridPage(2);
-      await tester.assertRowCountInGridPage(2);
       await tester.pumpAndSettle();
```

Comment:
Unrelated, but could the comment be fixed?

Reply:
What do you mean?